CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Project Pythia Cookbook Contributor's Guide
+
+Project Pythia Cookbooks are collections of more advanced and domain-specific example
+workflows building on top of [Pythia Foundations](https://foundations.projectpythia.org/landing-page.html). 
+They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
+and should direct the reader towards the [Foundations material](https://foundations.projectpythia.org/landing-page.html) for any required
+background knowledge. 
+
+The following is a step-by-step guide to getting your cookbook idea
+hosted on the [Project Pythia Cookbooks gallery](https://projectpythia.org/cookbook-gallery.html).
+
+1. Use the template
+    1. On https://github.com/ProjectPythiaTutorials/cookbook-template, click "Use this template"
+    1. Create the repo under your account with a descriptive name, followed by `-cookbook` (e.g. `hydrology-cookbook`, `hpc-cookbook`, `cesm-cookbook`, etc.)
+    1. In the settings, enable GitHub Pages and make the source `gh-pages/(root)`
+    1. See the [GitHub tutorial in Foundations](https://foundations.projectpythia.org/foundations/getting-started-github.html) for how to set up a clone
+1. Create the environment
+    1. Edit `environment.yml`: change the name to `<your-cookbook-name>-dev` (e.g. `cesm-cookbook-dev`) and add all required libraries and other dependencies under `dependencies:`. Commit the changes
+    1. Create the Conda environment with `conda env create -f environment.yml`. If it crashes, try running `conda config --set channel_priority strict`
+    1. Activate your environment with `conda activate <env-name>`
+1. Add content
+    1. Using the notebook template, add your content. Add folders to organize notebooks into sections if applicable
```

Comment:
Does this mean edit the file `notebooks/notebook-template.ipynb` directly to add content?

Reply:
Yes, added clarifying line